tools/add_font.py

```diff
@@ -132,21 +198,28 @@ def _WriteTextFile(filename, text):
 
 
 
-def main(argv):
-  if len(argv) != 2:
-    sys.exit('One argument, a directory containing a font family')
-  fontdir = argv[1]
+def main(args=None):
+  parser = ArgumentParser(description=__doc__)
```

Comment:
Please use gflags, as shown in sanity_check.py, to ensure we can sync the internal copy gracefully.